R/tm_missing_data.R

```diff
@@ -1393,3 +1405,7 @@ srv_missing_data <- function(id,
     ###
   })
 }
+
+missing_data_names <- c(
+  "Summary Obs", "Summary Patients", "Combinations Main", "Combinations Hist", "By Subject"
+)
```

Comment:
What about moving this to the top of the document?

Reply:
We can actually remove it. See e4ad8a26.

Variables are now in inline code for roxygen2 tag surrounded  by `nolint`